app/views/declaration/summary/related_documents.scala.html

```diff
@@ -0,0 +1,43 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import forms.declaration.Document
+@import models.requests.JourneyRequest
+@import models.viewmodels.HtmlTableRow
+@import services.DocumentType
+
+@(documents: Seq[Document])(implicit messages: Messages, journeyRequest: JourneyRequest[AnyContent])
+
+@if(documents.isEmpty) {
+    @components.table_row("previous-documents", HtmlTableRow(
+        label = messages("declaration.summary.transaction.previousDocuments"),
+        value = messages("site.no")
+    ))
+}
+
+@if(documents.nonEmpty) {
```

Comment:
else ?

Reply:
I stayed with speparated `if` to have more readable code here. I mean to see the condition when this part will appear, but probably it'll be better to use `else`